docs/changelog/131286.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 131286
+summary: Allow remote enrich after LOOKUP JOIN
+area: ES|QL
+type: bug
```

Comment:
I'd rather type it as `enhancement` - we're adding functionality here (relative to 9.1 and before).